src/__test__/__snapshots__/all.spec.js.snap

```diff
@@ -0,0 +1,132 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`all.js 1`] = `
+"console.log(a);
+console.log(a.length);
+console.log(a.map());
+
+function add() {
+  console.log(\\"1\\");
+}
+
+const subtract = () => {
+  console.log(\\"2\\");
+};
+
+let multiply;
+multiple = () => {
+  console.log(\\"3\\");
+};
+
+let divide;
+divide = function() {
+  console.log(\\"4\\");
+};
+
+const power = function pow() {
+  console.log(5);
+};
+
+const obj = {
+  method1: function() {
+    console.log(6);
+  },
+  method2: () => {
+    console.log(7);
+  },
+  method3() {
+    console.log(8);
+  },
+};
+
+const Component = () => {
+  const privateMethod = () => {
+    console.log(1);
+  };
+  return class {
+    showList = () => {
+      arr.map(i => {
+        console.log(i);
+      });
+    };
+    render() {
+      console.log(2);
+    }
+  };
+};
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\\"use strict\\";
+
+var _classCallCheck2 = require(\\"babel-runtime/helpers/classCallCheck\\");
```

Comment:
I would recommend against including any more plugins in your tests other than yours. Your snapshots will be flaky this way and it makes things harder to know which changes your plugin is making.